extensions/wikia/ArticleVideo/OoyalaConfigController.class.php

```diff
@@ -251,10 +251,7 @@ class OoyalaConfigController extends WikiaController {
 				'fontString' => 'l',
 				'fontStyleClass' => 'oo-icon oo-icon-discovery-binoculars',
 			],
-			'quality' => [
-				'fontFamilyName' => 'ooyala-slick-type',
-				'fontString' => 'm',
-				'fontStyleClass' => 'oo-icon oo-icon-bitrate',
+			'quality' => [// svg set in skin()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)